src/components/layouts/Footer.jsx

```diff
@@ -1,8 +1,13 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
+
+
 const Footer = () => {
   return (
-    <footer className="flex flex-row items-center text-center justify-center bg-lightGreen h-16 text-black shrink-0">
-      <p>CTD Practicum Team 4</p>
+    <footer className="flex items-center text-center justify-between bg-lightGreen h-16 text-black shrink-0 px-4 lg:px-16 xl:px-14 2xl:px-20">
+      <p>&copy; TalentStudio</p>
+      <p>Designed and developed by Team 4 for Code the Dream</p>
+      <Link to="/team" class="hover:text-white hover:underline">Meet the Team</Link>
```

Comment:
This link shouldn't be available when user is already on the team page.

Reply:
I removed the link but still left the text 'Meet the Team' Is that ok?

Reply:
Maybe changing it to the "The Team" would be better.

Reply:
I changed "Meet the Team" to "The Team"